blocks/api/registration.js

```diff
@@ -44,6 +44,19 @@ export function registerBlockType( name, settings ) {
 		);
 		return;
 	}
+	if ( ! settings ||
+			( settings && typeof settings.save !== 'function' ) ) {
```

Comment:
Minor: should we use `isFunction` from `lodash`? It handles a special case for us.